projects/1.18/assets/domestication-innovation/domesticationinnovation/lang/zh_cn.json

```
@@ -0,0 +1,115 @@
+{
+  "itemGroup.domesticationinnovation": "驯养革新",
+  "item.domesticationinnovation.collar_tag": "项圈标签",
+  "item.domesticationinnovation.rotten_apple": "烂苹果",
+  "item.domesticationinnovation.sinister_carrot": "阴恶胡萝卜",
+  "item.domesticationinnovation.deflection_shield": "偏转护盾模型",
+  "item.domesticationinnovation.magnet": "磁铁模型",
+  "item.domesticationinnovation.feather_on_a_stick": "羽毛钓竿",
+  "item.domesticationinnovation.deed_of_ownership": "宠物契约",
+  "item.domesticationinnovation.deed_of_ownership.desc": "同 %s 绑定",
+  "block.domesticationinnovation.pet_bed_white": "白色宠物床",
+  "block.domesticationinnovation.pet_bed_orange": "橙色宠物床",
+  "block.domesticationinnovation.pet_bed_magenta": "品红色宠物床",
+  "block.domesticationinnovation.pet_bed_light_blue": "淡蓝色宠物床",
+  "block.domesticationinnovation.pet_bed_yellow": "黄色宠物床",
+  "block.domesticationinnovation.pet_bed_lime": "黄绿色宠物床",
+  "block.domesticationinnovation.pet_bed_pink": "粉红色宠物床",
+  "block.domesticationinnovation.pet_bed_gray": "灰色宠物床",
+  "block.domesticationinnovation.pet_bed_light_gray": "淡灰色宠物床",
+  "block.domesticationinnovation.pet_bed_cyan": "青色宠物床",
+  "block.domesticationinnovation.pet_bed_purple": "紫色宠物床",
+  "block.domesticationinnovation.pet_bed_blue": "蓝色宠物床",
+  "block.domesticationinnovation.pet_bed_brown": "棕色宠物床",
+  "block.domesticationinnovation.pet_bed_green": "绿色宠物床",
+  "block.domesticationinnovation.pet_bed_red": "红色宠物床",
+  "block.domesticationinnovation.pet_bed_black": "黑色宠物床",
+  "block.domesticationinnovation.drum": "命令鼓",
+  "message.domesticationinnovation.command_0": "%s在游走",
+  "message.domesticationinnovation.command_1": "%s在停留 ",
+  "message.domesticationinnovation.command_2": "%s在跟随",
+  "message.domesticationinnovation.drum_command_0": "命令%s游走",
+  "message.domesticationinnovation.drum_command_1": "命令%s停留",
+  "message.domesticationinnovation.drum_command_2": "命令%s跟随",
+  "message.domesticationinnovation.respawn": "%s在它的床上重生了",
+  "message.domesticationinnovation.remove_respawn": "移除%s重生记录的床",
+  "message.domesticationinnovation.goodbye": "%s将不会重生，永别了……",
+  "message.domesticationinnovation.enchantments": "魔咒：",
+  "message.domesticationinnovation.set_owner": "%s现在是%s的主人了",
+  "entity.minecraft.villager.domesticationinnovation.animal_tamer": "驯兽师",
+  "entity.domesticationinnovation.chain_lightning": "闪电链",
+  "entity.domesticationinnovation.recall_ball": "召回宝盒",
+  "entity.domesticationinnovation.feather": "羽毛",
+  "entity.domesticationinnovation.following_jukebox": "浮动唱片机",
+  "enchantment.domesticationinnovation.health_boost": "额外生命",
+  "enchantment.domesticationinnovation.health_boost.desc": "每提升一附魔等级增加宠物最大生命值 10 点",

```

```suggestion
  "enchantment.domesticationinnovation.health_boost.desc": "每提升一魔咒等级增加宠物最大生命值10点。",
```
下方所有`附魔等级`如果意思相同请全部改为`魔咒等级`
以及魔咒描述末尾可以加个句号，~~*我也不知道为啥Enchantment Description里面都有句号*~~